src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -1008,15 +1008,12 @@ private unsafe String[] SplitInternal(char separator, int count, StringSplitOpti
 
         [ComVisible(false)]
         [System.Security.SecuritySafeCritical]
-        internal String[] SplitInternal(char[] separator, int count, StringSplitOptions options)
+        private unsafe String[] SplitInternal(char[] separator, int count, StringSplitOptions options)
```

Comment:
I could not find any uses of this outside of `String` in both CoreCLR and looking on http://referencesource.microsoft.com, so I don't see why it needs to be internal.

Could `[ComVisible(false)]` be removed?